content/en/blog/2024/fuzzing-audit-results.md

```diff
@@ -0,0 +1,72 @@
+---
+title: OpenTelemetry Collector Completes Fuzzing Audit
+linkTitle: Fuzzing Audit Results
+date: 2024-12-20
+author: '[Adam Korczynski](https://github.com/AdamKorcz)'
+issue: 5798
+sig: GC
+cSpell:ignore: Korczynski containerd
+---
+
+OpenTelemetry is happy to announce the completion of the Collector's fuzzing
+audit sponsored by [the CNCF](https://www.cncf.io/) and carried out by
```

Comment:
Catching up on this: @AdamKorcz - do you have a link that supports the "CNCF sponsored this" claim?

Reply:
@chalin Please send me an email at adam@adalogics.com and I can put you in touch with someone in the CNCF who can confirm this.